pkg/operation/care/health.go

```diff
@@ -318,7 +318,7 @@ func (h *Health) checkSystemComponents(
 		return &c, nil
 	}
 
-	if established, err := botanist.CheckTunnelConnection(ctx, h.shootClient, logrus.NewEntry(logger.NewNopLogger()), common.VPNTunnel); err != nil || !established {
+	if err := botanist.CheckAndWaitForTunnelConnection(ctx, h.shootClient, h.shoot, logrus.NewEntry(logger.NewNopLogger()), common.VPNTunnel, 2*time.Minute); err != nil {
```

Comment:
The health/care check is executed every minute (default). Now, we are waiting for max. `2m` here. Shouldn't this value be much lower, something like `10s`?

Reply:
I was also wondering about that. The pod usually takes a bit more than 10 seconds to start so probably 20-25 seconds timeout should be ok?

Reply:
Not sure, I have some troubles with hard-coding the value here since it is dependent on what a human operator has set in the gardenlet's component config.
What was again the motivation for the PR - shouldn't the check become "green" after at most `1m` anyways?

Reply:
The motivation was that sometimes it is annoying and/or hard to track real issues when there are lots of clusters and you are monitoring the dashboard for potentially broken shoots

Reply:
Are you talking about a specific use-case (integration tests creating new shoots and waiting for them to be ready, for example) or is this a general observation? For the latter, I'd think that there's indeed a problem with the tunnel if it this check fails.